sycl/include/sycl/aliases.hpp

```diff
@@ -82,17 +72,37 @@ class half;
 namespace sycl {
 __SYCL_INLINE_VER_NAMESPACE(_V1) {
 using byte __SYCL2020_DEPRECATED("use std::byte instead") = std::uint8_t;
+using half = sycl::detail::half_impl::half;
+
+namespace opencl {
+// Strictly speaking, those alases do not exist in this namespace in SYCL 1.2.1,
+// but we still provide them for the sake of implmentation simplicity
+using cl_bool = bool;
+using cl_char = std::int8_t;
+using cl_uchar = std::uint8_t;
+using cl_short = std::int16_t;
+using cl_ushort = std::uint16_t;
+using cl_int = std::int32_t;
+using cl_uint = std::uint32_t;
+using cl_long = std::int64_t;
+using cl_ulong = std::uint64_t;
+using cl_half = half;
+using cl_float = float;
+using cl_double = double;
```

Comment:
I think vector aliases are missing in the spec by accident and probably should be added back for consistency.
If so, will it be preferable to use aliases instead of vec class?

Reply:
> If so, will it be preferable to use aliases instead of vec class?

Aliases will certainly be shorter than direct use of `vec` class (at least if you don't have `using namespace ...`), but I don't think that it matters a lot for the implementation: SYCL 2020 is decoupled from OpenCL and at intel/llvm we have several non-OpenCL backends. Therefore, I think that we should use minimal amount of OpenCL-specific types in the implementation because there should be no need for them.